cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -967,6 +974,19 @@ func (a *StaticAutoscaler) obtainNodeLists(cp cloudprovider.CloudProvider) ([]*a
 	return allNodes, readyNodes, nil
 }
 
+func filterNodeGroups(cp cloudprovider.CloudProvider, nodes ...*apiv1.Node) ([]*apiv1.Node, caerrors.AutoscalerError) {
```

Comment:
nit: This method does not filter node groups, it filters nodes that have an associated node group. Please rename it so it better reflects this.

Reply:
I felt like based on the whole function signature it made sense, but sure I can make this even more explicit.